configs/dependency-specs/ubuntu16_04.yaml

```diff
@@ -5,7 +5,7 @@ revisionsFilePath: "configs/dependency-tracking/ubuntu16_04.yaml"
 gcrDeps:
   # The RBE Ubuntu 16.04 toolchain container.
   - name: "IMAGE"
-    location: "l.gcr.io/google/rbe-ubuntu16-04"
+    location: "marketplace.gcr.io/google/rbe-ubuntu16-04"
```

Comment:
did you double check that all the services / GCB builds that use this work properly when switching from l.gcr.io to marketplace.gcr.io?

Reply:
GCB should be able to pull from marketplace.gcr.io. There was issue before, but it has been solved. Will send you the internal thread.